### DISCUSSION TITLE
Cannot pass legends as a prop for Waffle chart

### DISCUSSION THREAD

**Discussion by yotsaphonSutweha** [plouc/nivo#2095] [Q&A]
Hello,

I'm trying to build a Waffle chart in my React project. I'm passing the legends prop to the waffle component and I got this error message "Property 'legends' does not exist on type 'IntrinsicAttributes & IntrinsicClassAttributes<WaffleHtml> & Readonly<WaffleBaseProps & ColorProps<WaffleDatum> & Partial<...> & Partial<...> & Dimensions> & Readonly<...>'." However, I saw on the example on nivo website with the prop being passed and the chart there is working fine.

My nivo/waffle version is 0.79.1 and my react version is 16.14.0. 

Is someone experiencing the same issue? Thank you in advance.

**Comment by thepauljones** [comment]
Yes, same problem. Pasted from the official example on the site, and the same error...
